boskos/reaper/reaper.go

```diff
@@ -43,31 +44,35 @@ func main() {
 	if len(rTypes) == 0 {
 		logrus.Fatal("--resource-type must not be empty!")
 	}
-
-	for range time.Tick(time.Minute * 5) {
+	frequency := (*expiryDuration) / 6
```

Comment:
this is pretty magical? I think just set it to 1min is fine

Reply:
done